Makefile

```diff
@@ -80,6 +80,13 @@ cf-create-cdn-route:
 	$(if ${DNS_NAME},,$(error Must specify DNS_NAME))
 	cf create-service cdn-route cdn-route document-download-cdn-route -c '{"domain": "${DNS_NAME}"}'
 
+.PHONY: cf-create-prometheus-service-broker
+cf-create-prometheus-service-broker:
```

Comment:
Maybe this would fit better into [here][https://github.com/alphagov/notifications-aws/tree/master/paas]?

Reply:
Yeah, good call. Removed this, will make a separate PR for aws.